### ISSUE TITLE
`file.warn` is `undefined`

### ISSUE THREAD

**Issue by piranna** [RichardLitt/remark-lint-appropriate-heading#7]
https://github.com/RichardLitt/remark-lint-appropriate-heading/blob/3259a6e7191f7115f0fa0f1e82aa7f866461b354/lib/appropriate-heading.js#L56

Seems dependencies of `standard-readme` and `standard-readme-cli` got obsolete and API has changed. There's now a `file.fail`but not sure if it has the same purposse.

```json
  "dependencies": {
    "remark": "^11.0.2",
    "standard-readme-preset": "^1.0.4",
    "to-vfile": "^6.0.0"
  },
```

**Comment by wooorm** [comment]
`file.message` is what you want in this case, that replaced `file.warn`!

**Comment by piranna** [comment]
Then we need to update the code here :-)

**Comment by piranna** [comment]
Fixed in #8, can you review and merge?

**Comment by piranna** [comment]
Thank you :-)
